source/gui/settingsDialogs.py

```diff
@@ -2781,7 +2781,12 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 			maxValue=100
 		)
 		self.bindHelpEvent("OtherAppVolume", self.appSoundVolSlider)
-		self.appSoundVolSlider.SetValue(config.conf["audio"]["applicationsSoundVolume"])
+		volume = config.conf["audio"]["applicationsSoundVolume"]
+		if 0 <= volume <= 100:
+		    self.appSoundVolSlider.SetValue(volume)
+		else:
+		    log.error("Invalid volume level: {}", volume)
+		    self.appSoundVolSlider.SetValue(50)  # Set to default or previous valid value

```

Comment:
please use tabs not mixed indentation

Reply:
can we fetch the default value automatically here here? 100 is the default not 50.